slam-libraries/viam-cartographer/src/io/read_PCD_file.cc

```diff
@@ -38,6 +38,12 @@ ReadFile::timedPointCloudDataFromPCDBuilder(std::string file_path,
 
     // KAT NOTE: The file name format for the pcd files is assumed to be, e.g.:
     // rplidar_data_2022-02-05T01_00_20.9874.pcd
+
+    // LEAVING COMMENTED UNTIL READY TO TEST
+    // double time_delta = ReadTimeFromFilename(
```

Comment:
Leaving this commented as I did not get the chance to test this fully with cartographer and am unsure if other spots  would need this